tests/kafkatest/services/templates/mirror_maker_consumer.properties

```diff
@@ -27,3 +29,9 @@ offsets.storage={{ offsets_storage }}
 {% if consumer_timeout_ms is defined and consumer_timeout_ms is not none %}
 consumer.timeout.ms={{ consumer_timeout_ms }}
 {% endif %}
+
+{% if new_consumer %}
```

Comment:
I was getting a warning without this setting, and I was thinking that the default which was having only range based assignor is as bad. But I agree. I'll see if my interceptors work without this change ported here.